core/drivers/crypto/hisilicon/include/hisi_cipher.h

```diff
@@ -0,0 +1,72 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022, HiSilicon Limited
+ */
+#ifndef __HISI_CIPHER_H__
+#define __HISI_CIPHER_H__
+
+#include <drvcrypt.h>
+#include <drvcrypt_cipher.h>
+
+enum C_ALG {
+	C_ALG_DES = 0x0,
+	C_ALG_3DES = 0x1,
+	C_ALG_AES = 0x2,
+	C_ALG_SM4 = 0x3,
+};
+
+enum C_MODE {
+	C_MODE_ECB	= 0x0,
+	C_MODE_CBC	= 0x1,
+	C_MODE_CFB	= 0x2,
+	C_MODE_OFB	= 0x3,
+	C_MODE_CTR	= 0x4,
+	C_MODE_CCM	= 0x5,
+	C_MODE_GCM	= 0x6,
+	C_MODE_XTS	= 0x7,
+	C_MODE_CBC_CS = 0x9,
```

Comment:
indentation inconsistency.
I suggest to replace all above indentation with a unique space char.